sdk/python/lib/pulumi/provider/provider.py

```diff
@@ -66,6 +66,22 @@ def __init__(
         self.failures = failures
 
 
+class InvokeResult:
+    """InvokeResult represents the results of a call to `Provider.invoke`."""
+
+    outputs: Inputs
```

Comment:
I wonder if this should be `Mapping[str, Any]` (and in the `__init__`) since the values aren't going to be `Output[T]`.

Reply:
Yeh that seems reasonable